cli/README.md

```diff
@@ -62,4 +62,21 @@ DESCRIPTION
   Compile Player DSL files into JSON
 ```
 
+## `player dsl validate`
+
+Runs isolated TSC compiler on authored Player DSL Typescript files.
```

Comment:
To avoid confusion, we should clarify here that were running the type checker, we're not actually running the compiler.